documentation/src/main/asciidoc/topics/ref_rest_caches.adoc

```diff
@@ -318,11 +318,54 @@ GET /rest/v2/caches/{cacheName}?action=keys
 |===
 |Parameter |Required or Optional |Value
 
-|`batch-size`
+|`limit`
+|OPTIONAL
+|Specifies the maximum number of keys to be streamed. A negative value will retrieve the keys. The default value is `-1`.
+|`batch`
+|OPTIONAL
+|Specifies the internal batch size when retrieving the keys. The default value is `1000`.
+|===
+
+[id='rest_v2_get_entries']
+= Retrieving All Entries from Caches
+Invoke `GET` requests to retrieve all the entries in a cache in JSON format.
+
+[source,options="nowrap",subs=attributes+]
+----
+GET /rest/v2/caches/{cacheName}?action=entries
+----
+
+.Request Parameters
+
+|===
+|Parameter |Required or Optional |Value
+
+|`metadata`
+|OPTIONAL
+|Specifies if each entry should contain the metadata. The default value is `false`.
+|`limit`
+|OPTIONAL
+|Specifies the maximum number of keys to be streamed. A negative value will retrieve the keys. The default value is `-1`.
+|`batch`
 |OPTIONAL
 |Specifies the internal batch size when retrieving the keys. The default value is `1000`.
 |===
 
+{brandname} provides a JSON response such as the following:
+
+[source,json,options="nowrap",subs=attributes+]
+----
+include::json_examples/rest_caches_all_entries.json[]
+----
+
+* `key` The key of the entry.
+* `value` The value of the entry.
+* `lifespan` Value in milliseconds. -1 indicated not lifespan time. The value will not be present if metadata is parameter is `false`.
+* `maxIdle` Value in milliseconds. -1 indicated not maxIdle time. The value will not be present if metadata is parameter is `false`.
+* `created` Value in milliseconds. -1 indicated not created time. The value will not be present if metadata is parameter is `false`.
+* `lastUsed` Value in milliseconds. -1 indicated not last used time. The value will not be present if metadata is parameter is `false`.
+* `expireTime` Value in milliseconds. -1 indicated not expiration time. The value will not be present if metadata is parameter is `false`.
```

Comment:
Same wordind needs to be changed in those descriptions. E.g.:

```suggestion
* `expireTime` Value in milliseconds, or -1 if the entry does not expire. The value will not be present if metadata is parameter is `false`.
```

Reply:
What exactly is the expire time for an entry?